data/json/items/armor.json

```diff
@@ -10006,12 +10006,29 @@
         "flags" : ["VARSIZE"],
         "coverage" : 90,
         "material_thickness" : 1
+    },{
+        "type" : "ARMOR",
+        "id" : "tunic",
+        "name" : "tunic",
+        "weight" : 70,
```

Comment:
Sleeveless tunic weighs 60 grams. I'm not sure how much tunics actually weigh IRL.